### PR TITLE
add slow docker app to debugging section

### PR DESCRIPTION
**Pull request by sascala**
## Description
Add https://github.com/mesosphere/marathon/blob/master/docs/docs/cfs.md to DC/OS site.

## Urgency
- [ ] Blocker 
- [X] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, and 1.10).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.10/monitoring/debugging/slow-docker.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+menu_order: 40
+---
+
+If you recently upgraded to DC/OS 1.10 or configured `MESOS_CGROUPS_ENABLE_CFS=true` in your Mesos agent configuration, you may see slow-running Docker applications or slow deployments.
+
+# Strict CPU limitations are now default
+
+When using Apache Mesos, you can choose to use either CPU shares or strict CPU limitations. If you use CPU shares, your task can consume more CPU cycles than initially configured in your Marathon app definition, if your host system has free CPU cycles. If you are using strict CPU limitations, your task can only consume a maximum of CPU time based on your Marathon configuration.
+
+The default Mesos configuration uses CPU shares. CFS (Completely Fair Scheduler) strict CPU limitation was introduced as a default in DC/OS a while ago, but until recently this configuration is only respected by the Mesos executor and not by the Docker executor. [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) was fixed in the latest Mesos release, which is also included in DC/OS 1.10.
```
Comment (stbof) [inline 1.10/monitoring/debugging/slow-docker.md]: I'm used to introducing an acronym after the term:  Completely Fair Scheduler (CFS)

I suggest removing the "a while ago", as I don't think it adds anything.

Suggestion for the last sentence:

Completely Fair Scheduler (CFS) strict CPU limitations is the default in DC/OS, but until recently this configuration was respected only by the Mesos executor and not by the Docker executor. The fix [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) in the latest Mesos release and also included in DC/OS 1.10 removes this limitation.

File: 1.10/monitoring/debugging/slow-docker.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Your services or deployments are likely running slowly because they require more CPU cycles than they are configured to consume.
+
+# Steps to take
+
+## Increase CPU allocation
+
+If you have slow-running Docker services or deployments due to DC/OS upgrade or configuring `MESOS_CGROUPS_ENABLE_CFS=true`, increase the required CPU amount in your service definition. [From the CLI](/1.10/deploying-services/update-user-service/) or the **Services** tab of the DC/OS GUI, change the `"cpus"` property of your service definition to a higher value and test if this change solves your issues.
```
Comment (unterstein) [inline 1.10/monitoring/debugging/slow-docker.md]: I would make it a little bit more obvious, that we advocating to use this step to increase cpu allocation.

File: 1.10/monitoring/debugging/slow-docker.md (hunk 1 above)
Comment (sascala) [reply 1.10/monitoring/debugging/slow-docker.md]: 👍

File: 1.10/monitoring/debugging/slow-docker.md (hunk 2 above)
Comment (sascala) [reply 1.10/monitoring/debugging/slow-docker.md]: 👍

**Pull request merged**